src/components/analyses/landing/AnalysisSubmissionLanding.js

```diff
@@ -0,0 +1,620 @@
+            ...getAnalysisShareWithSupportRequest(
+                config?.analysis?.supportUser,
+                analysis.id
+            ),
+            supportRequest: analysisSupportRequest(
+                userProfile?.id,
+                userProfile?.attributes.email,
+                t("statusHelpRequestSubject", {
+                    name: userProfile?.attributes.name,
+                }),
+                comment,
+                analysis
+            ),
+        });
+    };
+
+    const { mutate: doTimeLimitExtension, isLoading: extensionLoading } =
+        useMutation(extendVICEAnalysisTimeLimit, {
+            onSuccess: (resp) => {
+                setConfirmExtendTimeLimitDlgOpen(false);
+                setTimeLimit(null);
+                //convert the response from seconds to milliseconds
+                announce({
+                    text: t("timeLimitExtended", {
+                        newTimeLimit: formatDate(resp?.time_limit * 1000),
+                        analysisName: analysis?.name,
+                    }),
+                    variant: SUCCESS,
+                });
+            },
+            onError: (error) => {
+                showErrorAnnouncer(t("analysesRelaunchError"), error);
+            },
+        });
+
+    const handleCancel = () => {
+        analysesCancelMutation({ id: analysis?.id });
+    };
+
+    const handleSaveAndComplete = () => {
+        const id = analysis?.id;
+        analysesCancelMutation({
+            id,
+            job_status: analysisStatus.COMPLETED,
+        });
+    };
+
+    const busy =
+        isFetching ||
+        isInfoFetching ||
+        isParamsFetching ||
+        analysisLoading ||
+        isFetchingTimeLimit ||
+        extensionLoading;
+
+    if (busy) {
+        return <GridLoading rows={25} baseId={baseId} />;
+    }
+
+    if (analysisFetchError) {
+        return (
+            <WrappedErrorHandler
+                errorObject={analysisFetchError}
+                baseId={baseId}
+            />
+        );
+    }
+
+    //if the analysis not found the a give id, analysis will be null/undefined.
+    if ((analysis === null || analysis === undefined) && !busy) {
+        return <ErrorTypography errorMessage={t("analysisNotFound")} />;
+    }
+
+    return (
+        <PageWrapper appBarHeight={75}>
+            <div
+                style={{ margin: theme.spacing(1), padding: theme.spacing(1) }}
+            >
+                {view === BATCH_DRILL_DOWN && <BackButton />}
+                <Grid container spacing={1}>
+                    <Grid item xs={isMobile ? 0 : 6}>
+                        <Grid container>
+                            <Grid item>
+                                <Typography variant="h6" color="primary">
+                                    {analysis?.name}
+                                </Typography>
+                                <AnalysisSubheader analysis={analysis} date={formatDate(analysis?.startdate)} />
```

Comment:
Might just be me, but when I was looking at the screenshots to see the running time, I initially couldn't find it.  I think I was expecting it to be part of the lower grid / form layout.  Not a blocker obviously, just something I thought I'd point out from my experience.

Reply:
With icons
![image](https://user-images.githubusercontent.com/1250790/140172207-2420384a-9318-4c4b-be7b-5984e6e5e4fb.png)

Reply:
![image](https://user-images.githubusercontent.com/1250790/140172615-fa8072b5-0a13-4f5e-a2da-b2459a6ac3e9.png)

Reply:
Green for completed makes sense to me and it's probably easier to change the status text color anyway 👍

Reply:
Mobile view with updated colors
![image](https://user-images.githubusercontent.com/1250790/140181551-6a21a25f-7b81-4adb-8874-0585f1b4928e.png)